packages/dds/test-dds-utils/src/ddsFuzzHarness.ts

```diff
@@ -1468,24 +1478,51 @@ export async function runTestForSeed<
 				),
 		  );
 	const summarizerClient = initialClient;
+	const handles = Array.from({ length: 5 }).map(() => uuid());
+	let handleGenerated = false;
 	const initialState: DDSFuzzTestState<TChannelFactory> = {
 		clients,
 		summarizerClient,
 		containerRuntimeFactory,
-		random,
+		random: {
+			...random,
+			handle: () => {
+				handleGenerated = true;
+				return new DDSFuzzHandle(
+					random.pick(handles),
+					// this is wonky, as get on this handle will always resolve via
+					// the summarizer client, but since we just return the absolute path
+					// it doesn't really matter, and remote handles will use
+					// the right handle context when they are deserialized
+					// by the dds.
+					//
+					// we re-used this hack a few time below, because
+					// we don't have the real client
+					initialState.summarizerClient.dataStoreRuntime,
```

Comment:
i did have an idea about making this better, specifically, override this in mixin client selection to use the real client. i'm not sure if we should keep this fall back or not though, we many need it for replay/minimization.

Reply:
you might even be able to make the below cases better by also doing them in mixin client selection, but again, not sure if we need the fall back. i also worry a bit about different modes running different logic, but i'm not sure the worry is founded.

Reply:
this could be done at a later time as well, as a change a this layer should be transparent to the tests themselves